.eslintrc.json

```diff
@@ -144,7 +125,6 @@
     "no-restricted-imports": ["error", "rxjs/Rx"],
     "no-shadow": "off",
     "no-throw-literal": "error",
-    "no-trailing-spaces": "error",
     "no-undef-init": "error",
     "no-underscore-dangle": "off",
     "no-unused-expressions": "off",
```

Comment:
Personally, I feel these removed rules are very helpful to maintaining a good formatted code base, especially, like no trailing spaces, delimiter, single quotes and so on. Instead of removing the rules, I think it would be better to add eslint disable, like`/* eslint-disable @typescript-eslint/member-ordering */` at the code that we intend to ignore the check due to a specific reason. If we indeed want to get rid of a rule, how about trying replace "error" with "off"? Then we could reset it when we need it again in the future.

Reply:
The reason I got rid of them is because we are using eslint in combination with prettier. We extend prettier in the eslint config, as described in the prettier docs here: https://prettier.io/docs/en/integrating-with-linters 
However there are a few rules in our config that pertain to style, not to linting and cause conflict between prettier and eslint. I have set to automatically format according to our prettierrc any time I save. But this then causes an error with eslint. According to the prettier docs found here: https://github.com/prettier/eslint-config-prettier you can run `npx eslint-config-prettier path/to/main.js` that will tell you which eslint rules conflict with prettier. So I removed those in the commit https://github.com/citrineos/citrineos-core/pull/113/commits/615d88b7af2e3ed0bac12bce8f2eee3f754caa9a